lib/omise/res/OmiseApiResource.php

```diff
@@ -222,16 +210,22 @@ protected function httpClient()
      */
     private function _executeCurl($url, $requestMethod, $key, $params = null)
     {
-        try {
-            $options = $this->getOptions($requestMethod, $key, $params);
-            $result = $this->httpClient()->request($requestMethod, $url, $options);
-            return $result->getBody();
-        } catch (ClientException $e) {
-            $array = json_decode($e->getResponse()->getBody()->getContents(), true);
-            throw OmiseException::getInstance($array);
-        } catch (Exception $e) {
-            throw new Exception($e->getMessage());
+        $ch = curl_init($url);
```

Comment:
is there a reason we're moving from Guzzle back to curl?

Reply:
We have merchants using legacy system and do not use composer. So, we decided to make minimal change and just make the codebase compatible from PHP 5.6 to latest.

We might put back Guzzle again in the future release once we start refactoring our codebase.